ballot/app/test/service.spec.ts

```diff
@@ -1,19 +1,54 @@
-import oasis from '@oasislabs/client';
+const oasis = require('@oasislabs/client');
 
 jest.setTimeout(20000);
 
-describe('Ballot', () => {
-  let service;
+describe('Ballot Test', () => {
+  let ballot;
 
   beforeAll(async () => {
-    service = await oasis.workspace.Ballot.deploy('Secret Ballot', ['Alice', 'Bob'], {
-      header: { confidential: false },
-      gasLimit: '0xe79732',
+    // Deploy your ballot service!
+    ballot = await oasis.workspace.Ballot.deploy('What\'s for dinner', ['beef', 'yogurt'], {
+      header: {confidential: false},
+      gasLimit: '0xf42400',
```

Comment:
you should revert my commit. I don't think this is necessary now that the oasis-chain bug has been resolved